docs/source/usecases/VirtualFunctions.md

```diff
@@ -111,6 +111,60 @@ deviceInstance->setAField(someHostValue); // set some field on the host
 
 This is the solution that the code teams we have talked to have said is the most productive way to solve the problem.
 
+## But what if I do not really need the V-Tables on the device side?
+
+There are cases where a lot of the logic of a code is implemented using virtual functions. Nevertheless, the performance critical part of the code might not use parent pointers and dynamic polymorphism to compute a result from data. In these cases, the device might not need to have a working virtual function table. Consider the following example:
```

Comment:
```suggestion
There are cases where a lot of the logic of a code is implemented using virtual functions. Nevertheless, the performance critical part of the code might not use pointers to base class and dynamic polymorphism to compute a result from data. In these cases, the device might not need to have a working virtual function table. Consider the following example:
```
First sentence seems unnecessary.

Reply:
I deleted the first sentence in [b0c6e4](https://github.com/kokkos/kokkos-core-wiki/pull/162/commits/b0c6e463f9c4bfbae4d48e1632851b4ec213cb89)